packages/editor/src/components/word-count/index.js

```diff
@@ -13,11 +13,7 @@ function WordCount( { content } ) {
 	 */
 	const wordCountType = _x( 'words', 'Word count type. Do not translate!' );
 
-	return (
-		<span className="word-count">
```

Comment:
Would it make sense to retain span wrapper here and only change the classname?

Reply:
I'd keep it decoupled. Adding the `.table-of-contents__number` class in there would make it tightly coupled with `table-of-contents`.